app/src/views/TermsAndConditions/ConditionsModal/index.tsx

```diff
@@ -0,0 +1,266 @@
+import { useMemo } from 'react';
+import {
+    Modal,
+    Table,
+} from '@ifrc-go/ui';
+import { SortContext } from '@ifrc-go/ui/contexts';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import { createStringColumn } from '@ifrc-go/ui/utils';
+import { isDefined } from '@togglecorp/fujs';
+
+import useFilterState from '#hooks/useFilterState';
+
+import i18n from './i18n.json';
+
+interface Props {
+    onCancel: () => void;
+}
+
+interface TableData {
+    id: number;
+    srcDbCode: string;
+    srcDbLab: string;
+    srcOrgCode: string;
+    srcOrgLab: string;
+    srcOrgtypeCode: string;
+}
+const keySelector = (d: TableData) => d.id;
+
+function ConditionalModal(props: Props) {
+    const strings = useTranslation(i18n);
+    const {
+        onCancel,
+    } = props;
+
+    const {
+        filtered,
+        sortState,
+    } = useFilterState<{
+        srcDbCode?: string,
+        srcDbLab?: string,
+        srcOrgCode?: string
+        srcOrgLab?: string
+        srcOrgtypeCode?: string
+    }>({
+        filter: {},
+    });
```

Comment:
Why do we need useFilterState without using the filters?

Reply:
For the sort, arigato!